app/src/main/java/mn/devfest/api/model/Session.java

```diff
@@ -102,6 +113,11 @@ public String getCategory() {
     }
 
     public DateTime getStartTime() {
-        return startTime;
+        try {
+            return DateTime.parse(startTime);
+        } catch (Exception e) {
+            Timber.e(e, "Failed to parse the startTime");
+            return new DateTime();
+        }
```

Comment:
Yeah, you're absolutely correct. That's definitely an artifact of me trying to "make it work" before I "make it good".
I'll change it now